common/commands/config.go

```diff
@@ -199,6 +203,17 @@ func (cc *ConfigCommand) config() error {
 	return config.SaveServersConf(configurations)
 }
 
+func (cc *ConfigCommand) isUrlUnsafe() bool {
+	artifactoryUrl := cc.details.Url
+	if !strings.HasPrefix(artifactoryUrl, "http://") {
+		return false
+	}
+	if strings.Contains(artifactoryUrl, "127.0.0.1") || strings.Contains(artifactoryUrl, "localhost") {
```

Comment:
I think that `strings.contains` isn't accurate enough for this check. For example, I can use a domain name that contains "localhost" but isn't "localhost", for example "localhost-123" and this test will pass. I therefore suggest to split the URL using a good utility, and then using `equals` to test this.